native/src/utils/DatabaseConnector.ts

```diff
@@ -226,19 +226,27 @@ class DatabaseConnector {
   }
 
   async migrationRoutine(): Promise<void> {
-    const contentDirExists = await BlobUtil.fs.isDir(CONTENT_DIR_PATH)
-    const baseContentDirExists = await BlobUtil.fs.isDir(UNVERSIONED_CONTENT_DIR_PATH)
-    const resourceCacheDirExists = await BlobUtil.fs.isDir(RESOURCE_CACHE_DIR_PATH)
-    const baseResourceCacheDirExists = await BlobUtil.fs.isDir(UNVERSIONED_RESOURCE_CACHE_DIR_PATH)
+    const contentDirExists = await RNFS.stat(CONTENT_DIR_PATH)
+      .then(stat => stat.isDirectory())
+      .catch(() => false)
+    const baseContentDirExists = await RNFS.stat(UNVERSIONED_CONTENT_DIR_PATH)
+      .then(stat => stat.isDirectory())
+      .catch(() => false)
+    const resourceCacheDirExists = await RNFS.stat(RESOURCE_CACHE_DIR_PATH)
+      .then(stat => stat.isDirectory())
+      .catch(() => false)
+    const baseResourceCacheDirExists = await RNFS.stat(UNVERSIONED_RESOURCE_CACHE_DIR_PATH)
+      .then(stat => stat.isDirectory())
+      .catch(() => false)
```

Comment:
🤔 Why not just use `exists()`?
```suggestion
    const contentDirExists = await RNFS.exists(CONTENT_DIR_PATH)
    const baseContentDirExists = await RNFS.exists(UNVERSIONED_CONTENT_DIR_PATH)
    const resourceCacheDirExists = await RNFS.exists(RESOURCE_CACHE_DIR_PATH)
    const baseResourceCacheDirExists = await RNFS.exists(UNVERSIONED_RESOURCE_CACHE_DIR_PATH)
```